content/concept-depositauth.md

```diff
@@ -0,0 +1,59 @@
+
+## Recommended Usage
+
+To get the full effect of Deposit Authorization, Ripple recommends also doing the following:
+
+- Always maintain an XRP balance higher than the minimum [reserve requirement](concept-reserves.html).
+- Keep the DefaultRipple flag in its default (disabled) state. Do not enable [rippling](concept-noripple.html) on any trust lines. When sending TrustSet transactions, always use the [`tfSetNoRipple` flag](reference-transaction-format.html#trustset-flags).
+- Do not place [Offers](reference-transaction-format.html#offercreate). It is impossible to know in advance which matching offers will be consumed to execute such a trade.
+
+## Precise Semantics
+
+An account with Deposit Authorization enabled:
+
+- **Cannot** be the destination of [Payment transactions][], with **one exception**:
+    - If the account's XRP balance is equal to or below the minimum account [reserve requirement](concept-reserves.html), it can be the destination of a Payment whose `Amount` is equal or less than the minimum account reserve (currently 20 XRP). This is to prevent an account from becoming "stuck" by being unable to send transactions but also unable to receive XRP. The account's owner reserve does not matter for this case.
+- Can receive XRP from [PaymentChannelClaim transactions][] **only if** the sender of the PaymentChannelClaim transaction is the destination of the payment channel.
+- Can receive XRP from [EscrowFinish transactions][] **only if** the sender of the EscrowFinish transaction is the destination of the escrow.
+- **Can** receive XRP or issued currencies by sending a [CheckCash][] transaction. _(Requires the [Checks amendment](reference-amendments.html#checks).)_
+- **Can** receive XRP or issued currencies by sending [OfferCreate transactions][].
+    - If the account sends an OfferCreate transaction that is not fully executed immediately, it **can** receive the remainder of the ordered XRP or issued currency later when the offer is consumed by other accounts' [Payment][] and [OfferCreate][] transactions.
+- If the account has created any trust lines without the [NoRipple flag](concept-noripple.html) enabled, or has enabled the DefaultRipple flag and issued any currency, the account **can** receive the issued currencies of those trust lines in [Payment transactions][] as a result of rippling. It cannot be the destination of those transactions.
+    - If the account has not created any trust lines with a nonzero limit, and has not issued currency on trust lines created by others, it **cannot** receive any issued currencies in any transaction. (This rule is not specific to the DepositAuth flag.)
```

Comment:
As you say, this statement is not related to DepositAuth.  But it's also not quite correct if I'm reading it right.  I can create an offer for a currency for which I have no trust line.  That offer can be crossed, and if it is, then I will have that currency even though I explicitly created no trust lines.  Offers are very powerful.